Formula/iam-policy-json-to-terraform.rb

```diff
@@ -0,0 +1,39 @@
+require "json"
+
+class IamPolicyJsonToTerraform < Formula
+  desc "Convert a JSON IAM Policy into terraform"
+  homepage "https://github.com/flosell/iam-policy-json-to-terraform"
+  url "https://github.com/flosell/iam-policy-json-to-terraform/archive/1.3.0.tar.gz"
+  sha256 "90b5be7caa232186718b5645b2d5ab65a59695854db7598462bdbe059444a051"
+  head "https://github.com/flosell/iam-policy-json-to-terraform.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+
+    dir = buildpath/"src/github.com/flosell/iam-policy-json-to-terraform"
+    dir.install buildpath.children
+    cd "src/github.com/flosell/iam-policy-json-to-terraform" do
+      system "go", "build", "-o", "iam-policy-json-to-terraform", "*.go"
+      bin.install "iam-policy-json-to-terraform"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    # test version
+    assert_match version.to_s, shell_output("#{bin}/iam-policy-json-to-terraform -version")
+
+    # test help
+    help_output = "Usage"
+    assert_match help_output.to_s, shell_output("#{bin}/iam-policy-json-to-terraform --help 2>&1", 2)
+
+    # test functionality
+    test_input = { :Statement => [{ :Effect => "Allow", :Action => ["ec2:Describe"], :Resource => "*" }] }
+    output = shell_output("echo '#{test_input.to_json}' | #{bin}/iam-policy-json-to-terraform")
+    assert_match "data", output
```

Comment:
Is that single word al that's guaranteed to be in the output?

Reply:
I tried doing an `assert_match` on multiple lines of output and it kept raising an error. Would you know the best way to verify multiline output?

```
✗ echo '{"Statement":[{"Effect":"Allow","Action":["ec2:Describe*"],"Resource":"*"}]}' | iam-policy-json-to-terraform
data "aws_iam_policy_document" "policy" {
  statement {
    sid       = ""
    effect    = "Allow"
    resources = ["*"]
    actions   = ["ec2:Describe*"]
  }
}
```

Reply:
I've changed it to check for `ec2:Describe*` instead which is a better check for the efficacy of the tool's output.